development/ts-migration-dashboard/common/build-module-partitions.ts

```diff
@@ -258,27 +288,27 @@ function buildModulesWithLevels(
       }
 
       // Skip files that weren't on the original list of JavaScript files to
-      // convert, as we don't want them to show up on the status dashboard
+      // convert, as we don't want them to show up on the dashboard
       if (
         !npmPackageMatch &&
-        !allowedFilePaths.includes(
-          dependencyModuleId.replace(/\.tsx?$/u, '.js'),
-        )
+        /\.jsx$/u.test(dependencyModuleId) &&
```

Comment:
This condition mistakenly excluded TypeScript files that were originally created (and thus were never converted) from showing up on the dashboard.